kubernetes/helm_charts/sunbird-RC/certificatesign/values.j2

```diff
@@ -35,3 +35,4 @@ certificatesignenv:
   CERTIFICATE_PRIVATE_KEY: |-
     {{ CERTIFICATE_PRIVATE_KEY | default("''") | indent(width=4) }}  
   SIGNING_KEY_TYPE: "{{ SIGNING_KEY_TYPE|default('RSA')}}"
+  CACHE_CONTEXT_URLS: "{{ CACHE_CONTEXT_URLS|default('https://{{sunbird_public_storage_account_name}}.blob.core.windows.net/{{sunbird_content_azure_storage_container}}/schema/v1_context.json,https://{{sunbird_public_storage_account_name}}.blob.core.windows.net/{{sunbird_content_azure_storage_container}}/schema/sunbird_context.json')}}"
```

Comment:
Please change this as below
```
CACHE_CONTEXT_URLS: "{{ cache_content_urls | default('https://' + sunbird_public_storage_account_name + '.blob.core.windows.net/' + sunbird_content_azure_storage_container + '/schema/v1_context.json,https://' + sunbird_public_storage_account_name + '.blob.core.windows.net/' + sunbird_content_azure_storage_container + '/schema/sunbird_context.json')}}"
```

Why?
1. Ansible will throw errors if variable definition is of this format `{{ XYZ: XYZ | default('ABC') }}`. LHS and RHS cannot use same variable names. So we change the case - `{{ XYZ: xyz | default('ABC') }}`.
2. We cannot have multiple template identifier like this `{{ {{ }} }}`. We can have only one template identifier and all variables will get replaced in it. Rest we should treat as strings in quotes and use concat operator - `{{ VAR1 + 'SOME TEXT' + VAR2 }}`

Reply:
updated the value as per the comment